packages/react-components/src/components/SendBoxErrorBar.tsx

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { MessageBar, MessageBarType } from '@fluentui/react';
+import React, { useEffect } from 'react';
+
+/**
+ * @internal
+ */
+export interface SendBoxErrorBarProps {
+  /** Error message to render */
+  message?: string;
+  /**
+   * Timeout in ms after which the error bar disappears.
+   * If undefined, the error bar will never disappear.
+   */
+  timeout?: number;
```

Comment:
Why do we remove the error message after a timeout? Do we do this to any other error messages?

Reply:
The reason is a mix of product and design requirements and mimics the file upload error behavior in teams.